devito/passes/iet/languages/CXX.py

```diff
@@ -0,0 +1,111 @@
+import numpy as np
+from sympy.printing.cxx import CXX11CodePrinter
+
+from devito.ir import Call, UsingNamespace, BasePrinter
+from devito.passes.iet.langbase import LangBB
+from devito.symbolics.extended_dtypes import c_complex, c_double_complex
+
+__all__ = ['CXXBB']
+
+
+def std_arith(qualifier=None):
+    if qualifier:
+        qual = qualifier if qualifier.endswith(" ") else f"{qualifier} "
+    else:
+        qual = ""
+    return f"""
```

Comment:
nitpicking: I'd move this inside a `complex.h` file and read its content into a global variable (or, maybe better, a memoized func) once and for all in this python file

Reply:
I like it like that it makes customizing it with inputs easier if needed

Reply:
what customization/inputs?

Reply:
ping

Reply:
ah, never mind, I think I might have just found out the answer